src/Core/src/Stream/ResponseBodyStream.php

```diff
@@ -10,6 +10,7 @@
  * Stream a HTTP response body.
```

Comment:
We should add a comment here and on `ResponseBodyResourceStream` so one would understand the difference between them.

Reply:
thanks